modules/pel/peltool/peltool.py

```diff
@@ -211,29 +211,41 @@ def severityBasedExclusion(uh: UserHeader, config: Config) -> bool:
             return False
     return True
 
-def excludePEL(uh: UserHeader, config: Config) -> bool:
+def includePEL(uh: UserHeader, config: Config) -> bool:
     """
     Evaluates whether a PEL meets criteria based on specified conditions.
-    Returns True if the entry should be Excluded, False otherwise.
+    Returns True if the entry should be Included, False otherwise.
     """
-    if config.critSysTerm and uh.eventSeverity != SeverityValues.critSysTermSeverity.value:
+    if config.critSysTerm and uh.eventSeverity == SeverityValues.critSysTermSeverity.value:
         return True
-    
-    if config.non_serviceable_only and uh.isServiceable():
+
```

Comment:
I think we decided on using the official definition of serviceable, which is:

        auto isHidden = true;
        if (((sevType != SeverityType::nonError) &&
             attr.actionFlags.test(reportFlagBit) &&
             !attr.actionFlags.test(hiddenFlagBit)) ||
            ((sevType == SeverityType::nonError) &&
             attr.actionFlags.test(serviceActionFlagBit)))
        {
            isHidden = false;
        }
(that's only called isHidden in the code because that's what the D-Bus property was named due to an early misunderstanding)

in text, that's - the PEL is serviceable if it isn't informational and the report flag is set and the hidden flag isn't set.  OR, it is informational but the serviceAction flag is set.

Reply:
Working on a patch for these changes.

Reply:
Created [PR59](https://github.com/ibm-openbmc/openpower-pel-parsers/pull/59) for above changes.